pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -181,6 +218,37 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 		return r.reconcileAndError(instance, err)
 	}
 
+	err, updateOwnedLeaseFailed := r.obtainLease(node)
+	if err != nil && updateOwnedLeaseFailed {
+		if instance.Status.ErrorOnLeaseCount == 0 {
+			log.Info("can't extend owned lease. uncordon for now")
+
+			// Uncordon the node
+			err = r.stopNodeMaintenanceImp(node)
+			if err != nil {
+				return r.reconcileAndError(instance,fmt.Errorf("Failed to uncordon upon failure to obtain owned lease : %v ", err))
+			}
```

Comment:
This code belongs in the block that checks `instance.Status.ErrorOnLeaseCount > MaxAllowedErrorToUpdateOwnedLease`

Reply:
done